tasks/common_nix.yml

```diff
@@ -1,10 +1,15 @@
 ---
 
 
-- name: common | Combine dictionary options
+- name: common | Combine dictionary options (web)
```

Comment:
Any reason why you split them ?

Reply:
From the 2nd commit in the series:

concourse_facts_local_users is set in tasks/auth.yml which is only
included on web nodes; therefore the combining step now fails on
worker-only nodes as that var is empty.